ctapipe/core/container.py

```diff
@@ -28,12 +28,18 @@ def __init__(self, default, description="", unit=None, ucd=None):
         self.ucd = ucd
 
     def __repr__(self):
-        desc = f'{self.description}'
+        desc = f"{self.description}"
         if self.unit is not None:
-            desc += f' [{self.unit}]'
+            desc += f" [{self.unit}]"
         return desc
 
 
+class DeprecatedField(Field):
```

Comment:
Shouldn't this raise a warning on something like `__getattr__`?

Reply:
yes, I like that suggestion better. I'll change it.

Reply:
Maybe only on `__get__`, as otherwise we will be swamped by warnings when setting those values in the event sources etc.

Sou only getting them should raise this warning, right?

Reply:
Oh, and this automatically removes these fields from `items` and `fields`, so they will not be saved...

Reply:
Mmh. forget it, this does not really work...

Reply:
Yeah, I think I"ll just revert to the original implementation, but add a warning string.